cfgov/unprocessed/css/on-demand/simple-chart-admin.less

```diff
@@ -0,0 +1,25 @@
+  @media (min-width: 970px) {
+  .struct-block.simple-chart-block {
+    display: flex;
+    flex-wrap: wrap;
+    margin-right: -1.5em;
+  }
+
+  .struct-block.simple-chart-block > .field {
```

Comment:
Is it necessary to have `.struct-block` in front of `.simple-chart-block`? Is just `.simple-chart-block` not enough specificity?